components/navigation/navigation.css

```diff
@@ -0,0 +1,21 @@
+.navbar{
+  background-color:var(--preferred-color);
+  display:flex;
+  padding:0.7rem;
+  border-top-left-radius: 0.3rem;
+  border-top-right-radius: 0.3rem;
+  
```

Comment:
There's only one code refactoring change that you can do it in future is that currently you have given direct values to all the css properties like padding and border radius. Try to declare a global variable in :root and use those global variables to use in properties.